addons/website_event_track_session/models/event_track.py

```diff
@@ -139,7 +139,10 @@ def _get_track_suggestions(self, restrict_domain=None, limit=None):
             lambda track:
                 (track.is_published,
                  track.is_track_live,
-                 track.track_start_remaining > 0,
+                 track.track_start_remaining == 0  # First get the tracks that started less then 10 minutes ago
+                    and track.track_start_relative < (10*60),
+                 track.is_track_soon,  # Then the one that will begin soon (30 minutes)
```

Comment:
is it necessary, as we have -1 * track.track_start_remaining ? They should be ordered by begin time. Or you just want to have all coming soon at the same level in sort and not ordered by starting minute ?

Reply:
indeed.